app/boards/arm/glove80/glove80_rh.dts

```diff
@@ -37,6 +37,24 @@
     vbatt: vbatt {
         compatible = "zmk,battery-nrf-vddh";
     };
+
+/*
+  MoErgo 40 LEDs
+
+         10 16 22 28 34
+       6 11 17 23 29 35
+       7 12 18 24 30 36
+       8 13 19 25 31 37
+       9 14 20 26 32 38
+         15 21 27 33 39
+  2 1 0
+  5 4 3
+*/
+
+    underglow_indicators: underglow-indicators {
+        compatible  = "zmk,underglow-indicators";
+        // Default: no indicators. Only `bat-self` is supported at the moment.
```

Comment:
Thought I'd stub this out for discoverability. Per the PR description, this is actually kind of a trap, because it doesn't work with the default definition of the magic key without pulling in an upstream bugfix.